tests/libmigration/migration.go

```diff
@@ -37,6 +37,22 @@ import (
 
 const MigrationWaitTime = 240
 
+func NewRandomMigration(vmiName string, namespace string) *v1.VirtualMachineInstanceMigration {
```

Comment:
Can we drop the "Random" bit? This function is very systematic.

Reply:
P.s. is this commit new? I don't recall it in former revisions of this PR. Since it touches so many files, it makes sense to move it to a new PR, so as not to block the merge of this. Also, can you add content to the commit message?

Reply:
If we want to move it in a separate PR then I will drop the commit.

Reply:
doh

Reply:
> doh

If it wasn't clear, I indeed confused unrelated PRs. My recommendation to review this commit elsewhere stands, but it's your call. It's mainly about the probability of git rebase conflicts.

Reply:
Thanks! Let's try to give it a try! If several `needs-rebase` will occur due to that change I will drop the commit. WDYT?

Reply:
> Ok for the name!

care to consider a shorter name?

Reply:
Missed change xD
Done!
Thanks